cephfs/admin/timestamp.go

```diff
@@ -0,0 +1,39 @@
+// +build !luminous,!mimic
+
+package admin
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// golang's date parsing approach is rather bizarre
```

Comment:
yes, it is! I assume this is in UTC?

Reply:
Yup. I double checked that's the Go default and what Ceph gives us. If it ever changed we could use other methods from the go stdlib (but it still uses that wacky "template" approach).